cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/model/objectstorage/ObjectStorageValidateRequest.java

```diff
@@ -24,6 +24,8 @@
 
     private String logsLocationBase;
 
+    private AzureParameters azure;
```

Comment:
Don't you want to include AzureParameters in a CloudParamters class, for cleaner future extensibility or you'd prefer to put any provider-specific param here in the future?

Reply:
I think it's not necessary here, there will be parameters related only to one provider at a time.